app/controllers/application_controller.rb

```diff
@@ -1,3 +1,3 @@
 class ApplicationController < ActionController::Base
-    before_action :authenticate_user! 
+    skip_before_action :verify_authenticity_token
```

Comment:
@moizansari22 Why you are not authenticating user?

Reply:
im authentication user of just VideoController not for all like API's . so i dont think so that 'before_action :authenticate_user!' should be in Application controller